recipes/libnoise/all/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+cmake_minimum_required(VERSION 3.4)
+project(noise)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+set(NOISE_SRC_DIR ${CMAKE_CURRENT_SOURCE_DIR}/source_subfolder/noise/src)
+
+file(GLOB NOISE_SRC_FILES
+    ${NOISE_SRC_DIR}/*.cpp
```

Comment:
The archive contains: `src/win32/libnoise.def`.
It contains a lot of symbols though.

So I think using `WINDOWS_EXPORT_ALL_SYMBOLS` is good enough.
As long as there are no static variables not exported.

Reply:
Yes you're right. And actually, looking at source code, all symbols are exported anyway. No static variables by the way.

Reply:
So let's keep it like it is.